tree/ntupleutil/v7/inc/ROOT/RNTupleInspector.hxx

```diff
@@ -212,7 +213,7 @@ public:
    ///
    /// \note Here, we assume that the compression settings are consistent across all clusters and columns. If this is
    /// not the case, an exception will be thrown when RNTupleInspector::Create is called.
-   int GetCompressionSettings() const { return fCompressionSettings; }
+   int GetCompressionSettings() const { return fCompressionSettings.value(); }
```

Comment:
Is there a reason why this shouldn't return an optional? 
The comment states that "we assume that the compression settings are consistent across all clusters and columns" - do we also assume the compression settings are not unknown? Perhaps we should add a brief explanation on why `fCompressionSettings` is an optional, it might not be obvious to a future reader.

Reply:
Returns now an `std::optional`